sentry_plugin/hooks/sentry_hook.py

```diff
@@ -4,64 +4,87 @@
 from airflow.models import TaskInstance
 from airflow.utils.db import provide_session
 
-from sentry_sdk import configure_scope, add_breadcrumb, init
+from sentry_sdk import (
+    configure_scope,
+    push_scope,
+    capture_exception,
+    add_breadcrumb,
+    init,
+)
 from sentry_sdk.integrations.logging import ignore_logger
 from sentry_sdk.integrations.flask import FlaskIntegration
 from sqlalchemy import exc
 
 SCOPE_TAGS = frozenset(("task_id", "dag_id", "execution_date", "operator"))
+SCOPE_CRUMBS = frozenset(
+    ("dag_id", "task_id", "execution_date", "state", "operator", "duration")
+)
 
 
 @provide_session
-def get_task_instance(task, execution_date, session=None):
+def get_task_instances(dag_id, task_ids, execution_date, session=None):
```

Comment:
This is a backwards incompatible change, think about your versioning.

Reply:
This plugin does not have any explicit version currently.